.schema-validator-version

```diff
@@ -0,0 +1 @@
+v3.11.0
```

Comment:
Can we add an extra line here since we do that in our version files in runner?

Reply:
Added an extra line to file.